collect_app/src/test/java/org/odk/collect/android/entities/EntitiesRepositoryTest.kt

```diff
@@ -437,6 +466,62 @@ abstract class EntitiesRepositoryTest {
         )
     }
 
+    @Test
+    fun `#getByAllByProperty returns entities without property when searching for empty string`() {
```

Comment:
This test and the one below do almost the same thing right?

Reply:
Kind of! There's a case where an entity is saved to `EntitiesRepository` without a property (the test on 470) and there's another where no entity for the list has that property (the tests at 496 and 511).